src/react-create-class.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const [major, minor] = React.version.split(".");
+
+const REACT15 = major === "15";
+const REACT155 = REACT15 && minor >= 5;
```

Comment:
I'm afraid these `const` and `let` won't work in our existing build process is simple and only transpiles imports exports...